reference/commands/consumer/config.rst

```diff
@@ -10,7 +10,7 @@ conan config
 
 .. code-block:: bash
 
-    $ conan config [-h] {rm,set,get,install} ...
```

Comment:
This shoudn't be part of this pr!

Reply:
yeah, it appeared after I just changed branch for PR, weird